helm/reana/templates/secrets.yaml

```diff
@@ -52,6 +52,6 @@ metadata:
     "helm.sh/resource-policy": keep
 type: Opaque
 data:
-  REANA_EMAIL_LOGIN: {{ .Values.notifications.email_config.login | b64enc }}
-  REANA_EMAIL_PASSWORD: {{ .Values.notifications.email_config.password | b64enc }}
+  REANA_EMAIL_LOGIN: {{ .Values.notifications.email_config.login | default "team@reana.io" | b64enc }}
+  REANA_EMAIL_PASSWORD: {{ .Values.notifications.email_config.password | default "123456" | b64enc }}
```

Comment:
Hmm, `123456` may evoke souvenirs in people. What about using `CHANGEME` or something as we discussed before?